src/main.c

```diff
@@ -277,6 +531,16 @@ void WINAPI EventCallback(PEVENT_RECORD ev)
         return;
     }
 
+    if (ev->EventHeader.EventDescriptor.Id == tidVMSwitchPacketFragment) {
+        AddVMSwitchPacketFragmentInfo = TRUE;
+        CreateVmSwitchPacketFragment(ev, LowerIfIndex);
+        // Change the ifIndex by VPortID for VmNIC and VlanId
+        //LowerIfIndex = VMSwitchPacketFragment.VmSwitchSourcePortId;
```

Comment:
What's this?

Reply:
The comment ? If yes, forgot to remove this.

Reply:
Yes, I was asking about the commented-out code. But also, it looks like the VlanId local variable is unused?